readalongs/cli.py

```diff
@@ -153,6 +152,7 @@ def cli():
 @click.option(
     "-t", "--text-grid", is_flag=True, help="Export to Praat TextGrid & ELAN eaf file"
 )
+@click.option("-H", "--html", is_flag=True, help="Export to WebComponent HTML")
```

Comment:
```suggestion
@click.option("-H", "--html", is_flag=True, help="Export to a single-file HTML format")
```
"WebComponent HTML" is not very meaningful to me.